tests/tracer/test_correlation_log_context.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+import structlog
+
+from ddtrace import Tracer
+from ddtrace import config
+from ddtrace import tracer
+from ddtrace.opentracer.tracer import Tracer as OT_Tracer
+from tests.utils import override_global_config
+
+
+@pytest.fixture
+def global_config():
+    config.service = "test-service"
+    config.env = "test-env"
+    config.version = "test-version"
+    yield
+    config.service = config.env = config.version = None
+
+
+def tracer_injection(logger, log_method, event_dict):
+    """Inject tracer information into custom structlog log"""
+    correlation_log_context = tracer.get_log_correlation_context()
+    # add ids and configs to structlog event dictionary
+    event_dict["dd"] = correlation_log_context
+    return event_dict
+
+
+def assert_dd_log_record_matches_context(dd_log_record, span, service, env, version):
```

Comment:
This logic just duplicates the logic done in the original method... kinda feels like we should be explicitly asserting these things

```python
assert dd_log_record == {
    "span_id": str(span1.span_id),
    "trace_id": str(span1.trace_id),
    "service": "test-service",
    "env": "test-env",
    "version": "test-version",
}
```